data/changelog/opam/2016-11-21-opam-lib-1-3.md

```diff
@@ -32,7 +32,7 @@ Therefore, I encourage all maintainers of tools based on opam-lib to migrate to
 1.3.
 
 The respective APIs are available in html for
-[1.2](https://opam.ocaml.org/doc/1.2/api) and [1.3](https://opam.ocaml.org/doc/1.3/api).
+[1.2](https://opam.ocaml.org/doc/1.2/api - [404 Not Found]) and [1.3](https://opam.ocaml.org/doc/1.3/api - [404 Not Found]).
```

Comment:
```suggestion
[1.2](https://opam.ocaml.org/doc/1.2/) and [1.3](https://opam.ocaml.org/doc/Upgrade_guide.html).
```
Since there isn't a link to the archived 1.3, I put in the Upgrade Guide so they can upgrade from 1.2 if necessary

Reply:
```suggestion
[1.2](https://web.archive.org/web/20210511004827/https://opam.ocaml.org/doc/1.2/api/) and [1.3](https://web.archive.org/web/20210511005041/https://opam.ocaml.org/doc/1.3/api/).
```